spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe '/quotes' do
+  it 'displays search results' do
+    query = 'I+am'
+    quotes = QuotesFacade.search_quotes(query)
+    quote = quotes.first
+
+    visit root_path
+
+    fill_in :keyword, with: query
+
+    click_on "Search for Quotes"
+    expect(page).to have_content("#{quotes.count} quotes found!")
+    expect(page).to have_content(quote.content)
+    expect(page).to have_content(quote.author)
+    expect(page).to have_content(quote.tags)
+  end
```

Comment:
This isn't necessarily testing that all quotes and their content/author/tags appear on the page. You could make a bit more specific of a test by saving the quote id from the API in the quote poro. Then, you could wrap each quote in a specific id. Doing that would let you write a test like this: 

```ruby
quotes.first(10).each do |quote|
      within ".quote-#{quote.id}" do 
        expect(page).to have_content(quote.content)
        expect(page).to have_content(quote.author)
      end 
```